src/cdef_apply.rs

```diff
@@ -536,9 +568,11 @@ pub(crate) unsafe fn rav1d_cdef_brow_16bpc(
                 edges as c_uint | CDEF_HAVE_LEFT as c_int as c_uint,
             );
         }
-        ptrs[0] = (ptrs[0]).offset((8 * PXSTRIDE((*f).cur.stride[0])) as isize);
-        ptrs[1] = (ptrs[1]).offset((8 * PXSTRIDE((*f).cur.stride[1]) >> ss_ver) as isize);
-        ptrs[2] = (ptrs[2]).offset((8 * PXSTRIDE((*f).cur.stride[1]) >> ss_ver) as isize);
+        ptrs[0] = (ptrs[0]).offset((8 * BD::pxstride((*f).cur.stride[0] as usize)) as isize);
+        ptrs[1] =
+            (ptrs[1]).offset((8 * BD::pxstride((*f).cur.stride[1] as usize) >> ss_ver) as isize);
+        ptrs[2] =
+            (ptrs[2]).offset((8 * BD::pxstride((*f).cur.stride[1] as usize) >> ss_ver) as isize);
```

Comment:
```suggestion
        ptrs[0] = (ptrs[0]).offset(8 * BD::pxstride((*f).cur.stride[0] as usize) as isize);
        ptrs[1] =
            (ptrs[1]).offset(8 * BD::pxstride((*f).cur.stride[1] as usize) as isize >> ss_ver);
        ptrs[2] =
            (ptrs[2]).offset(8 * BD::pxstride((*f).cur.stride[1] as usize) as isize >> ss_ver);
```

I've probably messed this up elsewhere myself, but signed/arithmetic and unsigned/logical shifts are different.

Reply:
indeed, good catch!